pkgs/tools/misc/cope/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitHub, perl, perlPackages, makeWrapper, }:
+
+perlPackages.buildPerlPackage rec {
+  pname = "cope";
+  version = "unstable-2021-01-29";
```

Comment:
Maybe `unstable-2015-01-29` if this is based on the commit timestamp? I am not sure.

I wonder if there are more actively developed forks or alternatives of this (not big issue though).